backend/api/users.py

```diff
@@ -57,7 +56,7 @@ def create_user():
 
 
 @users.route('/api/users', methods=['PUT'])
-@roles_required('admin')
+@login_required
```

Comment:
I am willing to merge this, but please open an issue then, because you should only be able to update your own profile and that is currently not enforced.